packages/core/src/Modules/Cashier/Containers/deposit.jsx

```diff
@@ -44,15 +44,19 @@ const Deposit = ({
     iframe_height,
     iframe_url,
     setActiveTab,
+    setDerivCrypto,
     onMount,
     container,
     currency,
     is_loading,
     setSideNotes,
 }) => {
+    const { is_deriv_crypto } = React.useContext(PlatformContext);
+
     React.useEffect(() => {
         setActiveTab(container);
         if (!is_virtual) {
+            setDerivCrypto(is_deriv_crypto);
```

Comment:
if we are setting this on cashier onload, is it needed again on deposit load?

Reply:
I was having second thoughts for this. Thanks for noticing this too. will remove and see if it works.